addons/website_sale_comparison/static/src/js/website_sale_comparison.js

```diff
@@ -244,6 +250,14 @@ var ProductComparison = publicWidget.Widget.extend(VariantMixin, {
             }
         }
     },
+    /**
+     * @private
+     *
+     * To hide popover on scroll.
+     */
+    _hideCompareButtonPopoverScroll() {
+        $('#comparelist .o_product_panel_header').popover('hide');
+    },
```

Comment:
The above jquery code could be replaced by following code : 
```suggestion
    _hideCompareButtonPopoverScroll() {
        const panelHeaderEl = document.querySelector("#comparelist .o_product_panel_header");
        Popover.getOrCreateInstance(panelHeaderEl).hide();
    },
```